bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigUtil.java

```diff
@@ -56,25 +60,31 @@ public static Object normalizeType(Object value) {
      * BigDecimals, Strings and Booleans wherever a conversion of similar types was possible.
      *
      * However, it does not check for general correctness of types. This can be done using the
-     * ConfigDescriptionValidator.
+     * {@link ConfigDescriptionValidator}.
      *
-     * @param configuration
-     * @param configDescription
-     * @return
+     * @param configuration the configuration to be normalized (if null then the given configuration is returned)
```

Comment:
What do you mean with "if null then the given configuration is returned"?
So, if configuration is null, then null is returned? But below you state that you return a non-null configuration.